toolchain/check/context.cpp

```diff
@@ -618,6 +618,28 @@ auto Context::ParamOrArgEnd(Parse::NodeKind start_kind) -> SemIR::InstBlockId {
   return ParamOrArgPop();
 }
 
+auto Context::FinalizeInits() -> void {
+  inst_block_stack().PushInit();
+  if (!inst_block_stack().PeekCurrentBlockContents().empty()) {
+    AddInst({Parse::NodeId::Invalid, SemIR::Return{}});
+    // Pop the Inits block here to finalize it
```

Comment:
```suggestion
    // Pop the Inits block here to finalize it.
```

Style nit: we use [standard punctuation](https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar) in comments.